NumLib/ODESolver/TimeDiscretization.h

```diff
@@ -114,9 +114,22 @@ class InternalMatrixStorage
 class TimeDiscretization
 {
 public:
+    TimeDiscretization() : _dx(nullptr) {}
+
     //! Sets the initial condition.
     virtual void setInitialState(const double t0, GlobalVector const& x0) = 0;
 
+    /*! Get the relative change of solutions between two successive time steps
+     *  by \f$ e_n = \|u^{n+1}-u^{n}\|/\|u^{n+1}\| \f$.
+     *
+     * \warning _x_old kept in the derived classes is changed to x - _x_old
+     *          after this computation! Therefore it can be only called just
+     *          before pushState(...), in which the value of _x_old is updated.
```

Comment:
This documentation is the comment for the initial implementation, and it is dropped now.